src/NUnitEngine/nunit.engine.tests/Internal/AddinsFileReaderTests2.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
+
+using NSubstitute;
+using NUnit.Engine.Internal.FileSystemAccess;
+using NUnit.Framework;
+using System.IO;
+
+namespace NUnit.Engine.Internal.Tests
+{
+    /// <summary>
+    /// Tests the implementation of <see cref="AddinsFileReader"/>.
+    /// </summary>
+    /// <remarks>All tests in this fixture modify the file-system. Therefore they need to be marked explicitly to run.</remarks>
+    [TestFixture, Explicit, Category("WritesToDisk"), NonParallelizable]
```

Comment:
If I recall correctly, we have a few other tests that write real files. I think we can enable this - my concern with it being explicit is that it would never get run! 😅

Reply:
Removed the 'explicit' attribute and the tests run fine on the build-servers. 😀